test/test-options.jl

```diff
@@ -17,6 +17,13 @@
     )
 end
 
+@testset "Test enable_names & direct_model usage" begin
+    connection = DBInterface.connect(DuckDB.DB)
+    _read_csv_folder(connection, joinpath(INPUT_FOLDER, "Tiny"))
+    energy_problem = TulipaEnergyModel.run_scenario(connection;)
+    TulipaEnergyModel.create_model!(energy_problem; enable_names = false, direct_model = true)
```

Comment:
@clizbe unless I am missing something, I think this can be simplified like this and still get the goal of the test:

```suggestion
    energy_problem = TulipaEnergyModel.run_scenario(connection; enable_names = false, direct_model = true)
```